FoGSE/visualization.py

```diff
@@ -1,18 +1,22 @@
 import sys, typing, logging, math
 import numpy as np
-from PyQt6 import QtCore, QtWidgets
+from PyQt6 import QtCore, QtWidgets, QtGui
 from PyQt6.QtCharts import QChart, QChartView, QLineSeries, QAbstractSeries
-from PyQt6.QtWidgets import QWidget, QPushButton, QRadioButton, QComboBox, QGroupBox, QLineEdit, QLabel, QGridLayout, QVBoxLayout, QHBoxLayout, QSpacerItem, QSizePolicy
+from PyQt6.QtWidgets import QWidget, QPushButton, QRadioButton, QComboBox, QGroupBox, QLineEdit, QLabel, QGridLayout, QVBoxLayout, QHBoxLayout, QSpacerItem, QSizePolicy, QTabWidget
 import pyqtgraph as pg
 
 from FoGSE.readBackwards import BackwardsReader
-import os
-
+import FoGSE.application
```

Comment:
Propose to remove this line to avoid circular importing which causes and error when testing. This isn't accessed in this module anyway so shouldn't cause an issue to remove.